### DISCUSSION TITLE
Undefined Quantum Gates -X, H, MZ when running CUDA-Q Docker in VS Code

### DISCUSSION THREAD

**Discussion by srinjoyganguly** [NVIDIA/cuda-quantum#1798] [Q&A]
Hello everyone, 

I am trying to run the Bernstein Vazirani algorithm using VS Code by running the latest CUDA-Q Docker image, but the file shows that the Quantum Gates - X, H and MZ are undefined. I am using Windows for the work. 

![cudaq_error1](https://github.com/NVIDIA/cuda-quantum/assets/35863175/b39bedcc-a7c4-4321-a990-7b22b5d12e4b)

Can you please help me figure out the problem? 

When I run the code, it shows a random output of a quantum circuit having six qubits. When I change the number of qubits, the output does not change and is not according to the number of qubits I gave. Here is the screenshot - 

![cudaq_error2](https://github.com/NVIDIA/cuda-quantum/assets/35863175/289d6599-901d-4a9b-8b0c-fe26465eeed3)

**Comment by khalatepradnya** [comment]
For Pylance error(s), can you try adding `from cudaq.qis import *` after the `import cudaq` statement?

**Comment by khalatepradnya** [comment]
The example you are trying doesn't take input argument `size` 
Ref: https://github.com/NVIDIA/cuda-quantum/blob/main/docs/sphinx/examples/python/bernstein_vazirani.py

Try changing `qubit_count` in the file.
